src/Event/Http/HttpRequestEvent.php

```diff
@@ -177,6 +177,28 @@ private function rebuildQueryString(): string
             return http_build_query($queryParameters);
         }
 
+        if (isset($this->event['requestContext']) && isset($this->event['requestContext']['elb'])) {
+            if (isset($this->event['multiValueQueryStringParameters'])) {
+                $queryParameters = $this->event['multiValueQueryStringParameters'];
+            } else {
+                $queryParameters = $this->event['queryStringParameters'] ?? [];
+            }
+
+            $queryString = '';
+
+            foreach ($queryParameters as $key => $values) {
+                $values = (array) $values;
+
+                foreach ($values as $value) {
+                    $queryString .= $key . '=' . $value . '&';
+                }
+            }
+
+            parse_str($queryString, $decodedQueryParameters);
+
+            return http_build_query($decodedQueryParameters);
+        }
```

Comment:
Hey @deleugpn, sorry for the delay. The PR looks good, thanks a lot!

One thing that could definitely help with maintenance later is being able to understand why these lines of code exist. Could you add some inline comments (in the code, not the PR) about "why"?

Here are some questions for example (that could be answered in comments):

- why do we need to handle ALB events separately here? (ideally you can add a link to the github issue or this PR)
- why the `foreach`?
- why the `parse_str`?

(try to word it so that someone else in 2 years will have to understand why some part of code exists ^^)